contractcourt/chain_watcher.go

```diff
@@ -57,6 +57,11 @@ type RemoteUnilateralCloseInfo struct {
 	CommitSet CommitSet
 }
 
+// BreachResolution wraps the outpoint of the breached channel.
+type BreachResolution struct {
+	FundingOutPoint wire.OutPoint
```

Comment:
That's true, this FundingOutPoint does not need to be here since in the `breachResolver` it will have access to the chanpoint through the resolver kit. I didn't know what else to put here. What would the bool store? I don't believe we can just store a nil value in bbolt because I would have done this

Reply:
I was thinking a version field in case it needs to change when we upgrade?

Reply:
I _think_ we could achieve that simply with a new TLV type? 

Either way we need to settle on what we want to store _now_. Funding outpoint here is fine, just that it doesn't give us any other unique information fwiw. 

We also have the option to use TLV _within_ the serialization as well which gives us a flexible upgrade path.

Reply:
Don't have a better option here. Initially was thinking storing a `1` might be enough and save us some space. Since a breach rarely happens, I think it's fine to leave it as is.

Reply:
I think that it's fine to store as-is, I would have liked to have caught this earlier in the process, but oh well.